custom/mmap_posix.go

```diff
@@ -30,7 +30,7 @@ func Mmap(f *os.File) ([]byte, error) {
 	}
 	p, err := syscall.Mmap(int(f.Fd()), 0, int(fi.Size()),
 		syscall.PROT_READ,
-		syscall.MAP_PRIVATE|syscall.MAP_DENYWRITE|syscall.MAP_POPULATE)
+		syscall.MAP_PRIVATE|0x800|0x8000)
```

Comment:
Why?

Reply:
Got undefined when building on my mac (seems the are platform dependant)
# github.com/tgulacsi/oracall/custom
custom/mmap_posix.go:33:31: undefined: syscall.MAP_DENYWRITE
custom/mmap_posix.go:33:53: undefined: syscall.MAP_POPULATE

Reply:
MAP_DENYWRITE is not what I thought of, MAP_POPULATE is just a slight optimization - just remove them!